README.md

```diff
@@ -1,5 +1,87 @@
-# HTTP Client Library
+# coreHTTP Client Library
 
 This repository contains a C language HTTP client library designed for embedded
```

Comment:
Missing perdiod after the first sentence.

Reply:
Thanks